// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript5/StrictMode/parserStrictMode8.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,3 +1,4 @@
 
 
 //// [/.src/parserStrictMode8.d.ts]
+declare function eval(): invalid;
